src/main/java/server/UserService.java

```diff
@@ -208,6 +208,11 @@ public void run() {
                             writeRoomStateToAllIn(roomState);
                             writeLobbyStateToAll();
                         }
+                        case Protocol.SEND_MESSAGE -> {
+                            String message = msgArr[2];
+                            List<String> userNames = roomStateRepository.requireRoomByUserName(msgArr[0]).getUserNames();
```

Comment:
144번 라인 보면 변수로 이미 치환되어있어서 이용하면 좋겠네요.
```suggestion
                            List<String> userNames = roomStateRepository.requireRoomByUserName(userName).getUserNames();
```